bootstrap-salt.sh

```diff
@@ -4538,6 +4538,13 @@ install_amazon_linux_ami_git_deps() {
         yum -y install ca-certificates || return 1
     fi
 
+    PIP_EXE='pip'
+    if [ $(command -v python2.7) ]; then
+        [ $(which pip2.7) ] || /usr/bin/easy_install-2.7 pip || return 1
```

Comment:
We have a special function: `__check_command_exists` (https://github.com/saltstack/salt-bootstrap/blob/develop/bootstrap-salt.sh#L118)
Quite useful! :smiley: